package ResusableFunction;

public class finallyCode {
	
	
	public static void main(String[] args) {
		try{
			System.out.println("Inside try...");
		}
		catch(Exception e){
			
		}
		finally{
			System.out.println("Inside Finally...");
		}
	}
	
	

}
